server/readme.md

```diff
@@ -4,4 +4,12 @@ To start server:
 
 2. Enter pypi environment by typing "pipenv shell"
 
```

Comment:
We might want to add a step to install the packages before hopping into the `python` repl.

I got the following error

> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gleek/Desktop/development/tests/students/zac-git-fix-me/server/app.py", line 1, in <module>
    from flask import Flask, request, jsonify
ModuleNotFoundError: No module named 'flask'

Reply:
We only need to use `pipenv install` and that will install all the packages that are listed in the `pipfile`